shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -471,17 +472,29 @@ - (void)surfaceUpdated:(BOOL)appeared {
   }
 }
 
+// Count of FlutterViews currently on-screen. Used to determine when to
+// swizzle/unswizzle status bar touch event handling.
+static int visibleFlutterViewCount = 0;
```

Comment:
Done, but then reverted after decision to go with manual registration in the delegate.